CLIENT-SPECIFICATION.md

```diff
@@ -1,15 +1,19 @@
 # tldr-pages client specification
 
- - **Current Specification Version:** 1.1
+ - **Current Specification Version:** 1.2
 
 This document contains the official specification for tldr-pages clients. It is _not_ a specification of the format of the pages themselves - only a specification of how a user should be able to interface with an official client.
 
 The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 
 ## Changelog
```

Comment:
Since this changelog is going to get longer and longer, I would suggest moving this section at the end of the file, referencing it in the first paragraph, like this (bold text added by me):

> This document contains the official specification for tldr-pages clients. It is _not_ a specification of the format of the pages themselves - only a specification of how a user should be able to interface with an official client. **For a list of previous versions of the specification, see the [changelog section](#Changelog) below.**

I would also suggest to link to specific versions of the file along with PR numbers, and have the list in reverse order (most recent first), like this (see source of this comment for the Markdown):

 - [v1.2, July 3rd 2019](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md) (#3168)
   - Added new `-L, --language` recommended command-line argument.

 - [v1.1, April 1st 2019](https://github.com/tldr-pages/tldr/blob/fbdc06b7425f92cc0d4fc9a5cfc5860ef017251e/CLIENT-SPECIFICATION.md) (#2859)
   - Clarified platform section.

 - [v1.0, January 23rd 2019](https://github.com/tldr-pages/tldr/blob/f5be8a2614a455288f26e42953efeb8cb3bc50b0/CLIENT-SPECIFICATION.md) (#2706)
   - Initial release.

Reply:
Done :D